LWCOSC/force-app/main/default/lwc/doPagination/doPagination.js-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>50.0</apiVersion>
+  <isExposed>true</isExposed>
```

Comment:
We should be mindful while "exposing" the component. If the component is to be used by other components only (as i thing is the case here) we should not expose this.

Reply:
Point Noted, As discussion No action require